### PR TITLE
Pol 179 1965254 dimension information missing

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
### Description

Date range calculation setting issue

### Issues Resolved

Date range calculation

### Contribution Check List

- [x] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

https://governance.rightscale.com/org/78/projects/30598/policy-incidents/5e3801b84ec7b3000163f0f9

image

### CODE REVIEW

File: cost/scheduled_reports/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0
```
Comment (cdwilhelm) [inline cost/scheduled_reports/CHANGELOG.md]: version should be 1.20.

File: cost/scheduled_reports/scheduled_report.pt [hunk 2]
```diff
@@ -20,7 +20,7 @@ category "Cost"
 default_frequency "monthly"
 tenancy "single"
 info(
-  version: "1.9",
+  version: "2.0",
```
Comment (cdwilhelm) [inline cost/scheduled_reports/scheduled_report.pt]: version should be 1.20.

File: cost/scheduled_reports/CHANGELOG.md (hunk 1 above)
Comment (gopinathmahakud1) [reply cost/scheduled_reports/CHANGELOG.md]: I have addressed the comment please review the changes

File: cost/scheduled_reports/scheduled_report.pt (hunk 2 above)
Comment (gopinathmahakud1) [reply cost/scheduled_reports/scheduled_report.pt]: I have addressed the comment please review the changes.

**Pull request merged**